stage0/src/main.rs

```diff
@@ -136,17 +140,12 @@ pub extern "C" fn rust64_start() -> ! {
     create_idt(idt);
     idt.load();
 
+    // We assume 0-th bit is never the encrypted bit.
```

Comment:
Somehow I think this assumption will bite us back when we may want to use the 0-th bit as the encrypted bit :)

Reply:
Bit 0 is the `PRESENT` bit, so I'd be impressed if they overload that one.

Reply:
ah, ok then, probably not :)